pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/LoadBalancerTest.java

```diff
@@ -321,8 +322,11 @@ public void testBrokerRanking() throws Exception {
         }
 
         for (int i = 0; i < BROKER_COUNT; i++) {
-            Method updateRanking = Whitebox.getMethod(SimpleLoadManagerImpl.class, "updateRanking");
-            updateRanking.invoke(pulsarServices[i].getLoadManager().get());
+            Method method = Whitebox.getMethod(SimpleLoadManagerImpl.class, "getUpdateRanking");
+            Object invoke = method.invoke(pulsarServices[i].getLoadManager().get());
```

Comment:
this line should be moved inside the `while` loop
other wise if `invoke` is null it will be an endless loop

Reply:
yes, thanks, right.